iparkerhodge/client-side-capstone
# Left Navigation

#Changes
-added Charts button to left nav
-tracked activeView state in MainContent to navigate between dashboard and charts view
-fixed toggleNav to only allow navigation use once the user is logged in

#To Test
-check that user cannot use navigation until logged in
-navigation between charts and dashboard is working
